lib/Db/MailAccount.php

```diff
@@ -186,7 +186,7 @@ public function __construct(array $params = []) {
 		$this->addType('inboundPort', 'integer');
 		$this->addType('outboundPort', 'integer');
 		$this->addType('lastMailboxSync', 'integer');
-		$this->addType('provisioned', 'bool');
+		$this->addType('provisioned', 'boolean');
```

Comment:
fyi `bool` and `boolean` should work: https://www.php.net/manual/en/function.settype.php

Reply:
Yes, we discussed this and concluded some consistency would be nice :)

Reply:
fine by me but weird to use boolean because for method signatures we need bool ;)